db/database.js

```diff
@@ -77,16 +86,16 @@ const createEnrollmentsTable = db => {
         REFERENCES cohorts(id),
       CONSTRAINT fk_student_id
         FOREIGN KEY (student_id)
-        REFERENCES students(id),
-      CONSTRAINT fk_enrollment_status
-        FOREIGN KEY (enrollment_status)
-        REFERENCES enrollment_status(id)
```

Comment:
I gave enrollment status as a string now and not referencing to enrollment_status table. Once we finalize the list of possible statuses, I will add back the reference

Reply:
Sounds good.